packages/p2p/src/components/buy-sell/buy-sell-header.jsx

```diff
@@ -32,24 +36,64 @@ const AnimationWrapper = ({ children, is_visible }) => {
 };
 
 const BuySellHeader = ({ is_visible, table_type, setTableType }) => {
+    const { buy_sell_store } = useStores();
```

Comment:
We're now observing data so we should wrap the component in `observer` (at the export level).